design/sd-local.md

```diff
@@ -38,6 +43,8 @@ test
 3. Validate screwdriver.yaml
    - Validate yaml
    - Get Template/Step information
+   - Merge Template／Step information and Meta／Secrets from sd-local options
+   - Format in the same format as payload from /v4/builds/{id}
```

Comment:
What is the objective of `Format`? 
I think maybe the steps information and it will become JSON format like the payload from /v4/build/{id}.

Reply:
Add JSON format.
Because the payload from `/v4/validator` is not this JSON format, `sd-local` needs to format the merged information.

Reply:
OK, I see.